dss/api/bundles/__init__.py

```diff
@@ -115,28 +113,102 @@ def put(uuid: str, replica: str, json_request_body: dict, version: str=None):
         BundleMetadata.CREATOR_UID: json_request_body['creator_uid'],
     }
 
-    # if it already exists, then it's a failure.
-    try:
-        handle.get_user_metadata(bucket, bundle_manifest_object_name)
-    except BlobNotFoundError:
-        # write manifest to persistent store
-        handle.upload_file_handle(
+    was_created = _save_object_if_dne(
```

Comment:
1. I prefer explicit names in function names rather than `dne`.
2. dne = does not exist?  i don't know if that fully describes its behavior.